README.md

```diff
@@ -177,25 +177,26 @@ class Program
                 cts.Cancel();
             };
 
-            while (!cts.IsCancellationRequested)
+            try
             {
-                try
-                {
-                    var cr = c.Consume(cts.Token);
-                    Console.WriteLine($"Consumed message '{cr.Value}' at: '{cr.TopicPartitionOffset}'.");
-                }
-                catch (ConsumeException e)
+                while (true)
                 {
-                    Console.WriteLine($"Error occured: {e.Error.Reason}");
-                }
-                catch (OperationCanceledException)
-                {
-                    break;
+                    try
+                    {
+                        var cr = c.Consume(cts.Token);
```

Comment:
note: throwing an `OperationCanceledException` when an operation is canceled is idiomatic.